notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>NIT: I would add "we had to generalize what we knew about transporting vectors on a Euclidean space"</p><p> </p><p>NIT: euclidean --&gt; Euclidean</p><p> </p><p>NIT: because they do not apply on manifolds --&gt; because they need to be generalized for manifolds.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>

Reply:
<p>:D</p>